bqplot/traits.py

```diff
@@ -145,6 +145,8 @@ def _set_value(self, value):
     def set(self, obj, value):
         #TODO: We shouldnt have to overload the set because
         # numpy doesn't support == comparions
+        #TODO: This should also be removed once _buffered_state_diff is
+        # fixed in ipywidgets
```

Comment:
I agree that there is an issue with `_buffered_state_diff`, but the fact that the numpy array emits an event on any assignment is a bug regardless.

Therefore, even when we have fixed the `_buffered_state_diff` thing, we should keep this.